### PR TITLE
Create 1908-01-23.xml

### PR DESCRIPTION
**Pull request by owenyanevich**

### CODE REVIEW

File: 1908-01-23.xml [hunk 1]
```diff
@@ -0,0 +1,2239 @@
...
+                </edition>
+            </editionStmt>
+            <publicationStmt>
+                <publisher>FSU University Libraries</publisher>
+                <pubPlace>Tallahassee, FL</pubPlace>
+            </publicationStmt>
+            <sourceDesc>
+                <bibl>
+                    <title>The Egyptian Gazette</title>
+                    <date when="1905-07-13">Thursday, January 23, 1908</date>
```
Comment (whanley) [inline 1908-01-23.xml:26]: update date attribute

**Review by whanley — COMMENTED** [review]
Great job--well done. This is a nice clean text.

On page 3, most of your `div`s should be `type="item"`. There are also some recurring features, which should be identified with attributes. Also, Local and General should be subdivided into items. More details here: https://dig-eg-gaz.github.io/issues/features/

**Review by owenyanevich — COMMENTED** [review]
Updating

**Comment by whanley** [comment]
Hi Owen, There is a problem with the filename in this pull request. You uploaded a file named `1908-1-23.xml` (i.e., without a zero before the month number), which did not replace `1908-01-23.xml`. Please consolidate all of your changes in a single file and remove the misnamed file, then send a new pull request.

**Pull request closed without merging**